Add memorial service (#326)
FamilySearch/GEDCOM: attribute-event-requests/INDI-event.md

| `FCOM` | 4.0 | First Communion | |
| `GRAD` | 4.0 | Graduation | |
| `IMMI` | 3.0 | [Immigration](#immigration) | |

> | | | [Memorial Service](#mmorial-service) | |

Author: ```suggestion
| | | [Memorial Service](#memorial-service) | |
```

In [GEDCOM X](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) with URI `http://gedcomx.org/Immigration`

## Memorial Service

### Description

An event to mark or remember the death or presumed death of an individual, such as a funeral, wake, celebration of life, or memorial service.

### Value

Found in the following historical records:

- Obituaries

> - Memorial announcments and programmes

Author: ```suggestion
- Memorial announcements and programs
- Church records
```

### Absence

The most closely related structures are:

- `DEAT`: expresses that an individaul died, which generally has a different date than the memorial.

> - `BURI`: describes what was done with the remains of the individaul, which may or may not conincide with a memorial serive.

Reviewer: ```suggestion
- `BURI`: describes what was done with the remains of the individual, which may or may not coincide with a memorial serive.
```
